File: app/src/main/java/br/com/jonathanzanella/myexpenses/helpers/CurrencyHelper.java
package br.com.jonathanzanella.myexpenses.helpers;

import java.text.NumberFormat;

public final class CurrencyHelper {
	private static final double TOTAL_CENTS = 100.0;
	private CurrencyHelper() {}

	public static String format(int cents) {
		return NumberFormat.getCurrencyInstance().format(cents / TOTAL_CENTS);
	}

	static String format(double cents) {
		return NumberFormat.getCurrencyInstance().format(cents / TOTAL_CENTS);
	}
}

File: app/src/main/java/br/com/jonathanzanella/myexpenses/helpers/CurrencyHelper.kt
package br.com.jonathanzanella.myexpenses.helpers

import java.text.NumberFormat

object CurrencyHelper {
    private val TOTAL_CENTS = 100.0

    fun format(cents: Int): String {
        return NumberFormat.getCurrencyInstance().format(cents / TOTAL_CENTS)
    }

    internal fun format(cents: Double): String {
        return NumberFormat.getCurrencyInstance().format(cents / TOTAL_CENTS)
    }
}
